geode-core/src/main/java/com/gemstone/gemfire/internal/security/GeodeSecurityUtil.java

```diff
@@ -471,16 +499,27 @@ public static PostProcessor getPostProcessor() {
     return postProcessor;
   }
 
-  public static boolean isClientSecurityRequired() {
-    return isClientAuthenticator || isIntegratedSecurity;
+  public static boolean isIntegratedSecurity(){
```

Comment:
We can get rid of this public method now. I believe now all the other components are just using component specific security enabled function.